src/main/java/org/apache/commons/lang3/builder/CompareToBuilder.java

```diff
@@ -600,10 +600,10 @@ public CompareToBuilder append(final boolean lhs, final boolean rhs) {
         if (lhs == rhs) {
             return this;
         }
-        if (!lhs) {
-            comparison = -1;
-        } else {
+        if (lhs) {
             comparison = +1;
```

Comment:
Why do we need `+` here?

Reply:
Actually, the plus sign is redundant here. I've tried to link commits with only one type of changes and therefore haven't removed it.